packages/lib-react-components/src/ZooniverseLogo/ZooniverseLogo.stories.js

```diff
@@ -0,0 +1,29 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Button, Grommet } from 'grommet'
+
+import readme from './README.md'
+import ZooniverseLogo from './ZooniverseLogo'
+
+export default {
+  title: 'Components/ZooniverseLogo',
+  component: ZooniverseLogo,
+  args: {
+    id: 'The Zooniverse',
```

Comment:
Should this be a HTML element `id`, in which case 'The Zooniverse' is not a valid identifier.